hadoop/src/main/java/org/apache/carbondata/hadoop/CarbonInputSplit.java

```diff
@@ -138,6 +138,19 @@ public CarbonInputSplit(String segmentId, Path path, long start, long length, St
     version = CarbonProperties.getInstance().getFormatVersion();
   }
 
+  public CarbonInputSplit(String segmentId, Path path, long start, long length,
```

Comment:
This is not required. can use above constructor and pass FileFormat.COLUMNAR_V3 for fileFormat

Reply:
ok